dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -387,67 +386,70 @@ internal class DokkaSymbolVisitor(
      *   only if [includeStaticScope] is enabled
      *
      * @param includeStaticScope a flag to add static members, e.g. `valueOf`, `values` and `entries` members for Enum.
-     * See [org.jetbrains.kotlin.analysis.api.components.KaScopeProvider.getStaticDeclaredMemberScope] for what a static scope is.
+     * See [org.jetbrains.kotlin.analysis.api.components.KaScopeProvider.staticDeclaredMemberScope] for what a static scope is.
      */
     @OptIn(KaExperimentalApi::class) // due to getSyntheticJavaPropertiesScope
     private fun KaSession.getDokkaScopeFrom(
-        namedClassOrObjectSymbol: KaNamedClassOrObjectSymbol,
+        namedClassOrObjectSymbol: KaNamedClassSymbol,
         dri: DRI,
         includeStaticScope: Boolean = true
     ): DokkaScope {
         // getCombinedMemberScope additionally includes a static scope, see [getCombinedMemberScope]
         // e.g. getStaticMemberScope contains `valueOf`, `values` and `entries` members for Enum
-        val scope = if(includeStaticScope) namedClassOrObjectSymbol.getCombinedMemberScope() else namedClassOrObjectSymbol.getMemberScope()
+        val scope = if(includeStaticScope) namedClassOrObjectSymbol.combinedMemberScope else namedClassOrObjectSymbol.memberScope
         val constructors = scope.constructors.map { visitConstructorSymbol(it) }.toList()
 
-        val callables = scope.callables.toList()
+        val callables = scope.callables
```

Comment:
same here about converting sequence to list, or we will recreate the sequence 3 times